### PR TITLE
check whether a string is palindrome

### LINKED ISSUE #1718: Pallindromes in Java/Python
Hi! I want to contribute by writing code for this program. It will:
1. Check if a number is palindrome or not
2.Check if a name/word is palindrome or not
Plz assign me this as a participant of GSSOC20.
@SakshiShreya @singh-shreya6 @asha15 @raghav-dalmia

### PR DESCRIPTION
**Pull request by raksha009**
Issue no #1718

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added a new code
- 

#### Languages Used:
- Python
- 

#### Files Added:
- palindrome_string.py
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
your code has plagiarism
so we are closing this PR and unassigned you from the issue

File: Palindrome/palindrome_string.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Python program to find whether a string is a palindrome or not
+
+# Function to reverse a String
+def reverse(str):
+
+    # Initialize variable
+    rev=""
+
+    for i in range (len(str)-1,-1,-1):
```
Comment (asha15) [inline Palindrome/palindrome_string.py]: keep a space after comma

File: Palindrome/palindrome_string.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        rev = rev + str[i]
+    return rev    # Returning reversed number
+
```
Comment (asha15) [inline Palindrome/palindrome_string.py]: remove this line

File: Palindrome/palindrome_string.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    
+# --- main --- 
+string=raw_input(("Enter a string: "))  #User Input
```
Comment (asha15) [inline Palindrome/palindrome_string.py]: keep space around =

File: Palindrome/palindrome_string.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+a  = reverse(string) #Function call
+
+if(a==string):  #Comparing the reversed String with original String
```
Comment (asha15) [inline Palindrome/palindrome_string.py]: keep space around ==

Comment (raksha009) [reply Palindrome/palindrome_string.py]: @asha15 made the changes. Plz review

**Pull request merged**